src/ol/css.js

```diff
@@ -54,7 +54,7 @@ _ol_css_.CLASS_CONTROL = 'ol-control';
  * @param {string} The CSS font property.
  * @return {Object.<string>} The font families (or null if the input spec is invalid).
  */
-_ol_css_.getFontFamilies = (function() {
+export var getFontFamilies = (function() {
```

Comment:
`export function` instead of `export var` ?

Reply:
I stumbled over this as well, here we are using an IIFE, so we can cache certain things. I wasn't sure how to handle this. simpy changing as you suggested raises an 'Expression expected' error. What do you suggest?

IMO `var` isn't technically false, is it?

Reply:
Since things are passing with this, I am tempted to merge this.

Reply:
yes, please merge